modules/dynamic-operator/config/crd/bases/kuberlogic.com_kuberlogicservices.yaml

```diff
@@ -73,6 +73,11 @@ spec:
               advanced:
                 description: any advanced configuration is supported
                 x-kubernetes-preserve-unknown-fields: true
+              archived:
+                default: false
+                description: Archived field allows to remove namespace and all related
```

Comment:
description: `Service namespace is removed when it is archived`

Reply:
Description is changed.